app/src/main/java/br/digitalhouse/marveltime/LoginActivity.java

```diff
@@ -0,0 +1,160 @@
+        Spannable mySpannable = (Spannable)loginRegistro.getText();
+
+        ClickableSpan myClickableSpan = new ClickableSpan() {
+            @Override
+            public void onClick(View widget) {
+
+                notificacaoParaProximaTela(); }
+        };
+        mySpannable.setSpan(myClickableSpan, i1, i2 + 1, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+    }
+
+    private void initViews (){
+        loginUsuario = findViewById(R.id.loginUsuario);
+        loginSenha = findViewById(R.id.loginSenha);
+        bntLogin = findViewById(R.id.btnLogin);
+        loginRegistro = findViewById(R.id.loginRegistro);
+    }
+
+
+    //INGHRIDY SANTOS  - 04/02/20 - BDEV02 - VALIDAÇÃO USUARIO
+    private boolean usuarioValido (String usuario){
+        if (usuario.contains("@") && usuario.contains(".com") )
+            return true;
+        else
+            return false;
+    }
+
+    private boolean usuarioVazio (String usuario){
+        if (usuario.isEmpty())
+            return true;
+        else
+            return false;
+    }
+
+
+    //INGHRIDY SANTOS  - 04/02/20 - BDEV03 - VALIDAÇÃO SENHA
+    public static boolean senhaValida(String senha) {
+        if (senha.length() < 6)
+            return false;
+
+        boolean achouNumero = false;
+        boolean achouMaiuscula = false;
+        boolean achouMinuscula = false;
+        boolean achouSimbolo = false;
+
+        for (char c : senha.toCharArray()) {
+            if (c >= '0' && c <= '9')
+                achouNumero = true;
+            else if (c >= 'A' && c <= 'Z')
+                achouMaiuscula = true;
+            else if (c >= 'a' && c <= 'z')
+                achouMinuscula = true;
+            else
+                achouSimbolo = true;
+        }
+        return achouNumero && achouMaiuscula && achouMinuscula && achouSimbolo;
+    }
+
+    private boolean senhaVazia (String senha){
+        if (senha.isEmpty())
+            return true;
+        else
+            return false;
+    }
+
+    //INGHRIDY SANTOS  - 04/02/20 - BDEV02 - NOTIFICAÇÕES
+
+    protected void notificacao (){
+        Context contexto = getApplicationContext();
+        String textoNotificacao = "Por favor, preencha todos os campos";
+        int duracaoNotifacao = Toast.LENGTH_LONG;
+
+        Toast toast = Toast.makeText(contexto, textoNotificacao, duracaoNotifacao);
+        toast.show();
+    }
+
+    protected void notificacaoInformacaoIncorreta (){
+        Context contexto = getApplicationContext();
+        String textoNotificacao = "Usuário ou senha não atendem as regras, por favor, tente novamente";
+        int duracaoNotifacao = Toast.LENGTH_LONG;
+
+        Toast toast = Toast.makeText(contexto, textoNotificacao, duracaoNotifacao);
+        toast.show();
+    }
+
+    protected void notificacaoParaProximaTela (){
+        Context contexto = getApplicationContext();
+        String textoNotificacao = "PARABÉNS, em instantes você será direcionado a próxima tela.";
```

Comment:
Uma sugestão para essa notificação pensando na experiência do usuário, seria tirar as palavras em instantes, pois pode passar para o usuário a impressão de que esta ação irá demorar.

Reply:
Na verdade essa notificação de próxima tela deve ser tirada por quem for fazer a transição entre as telas. 
Como no meu codigo não contempla a próxima tela, então para testar se o botão está funcionando e deixar um "alerta" de que isso deve ser mudado, fiz uma notificação.

Reply:
Ok, entendido!